pkgs/tools/archivers/torrent7z/default.nix

```
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, fetchpatch, lib, ncurses }:
+
+stdenv.mkDerivation rec {
+  pname = "torrent7z";
+  version = "1.3";
+
+  src = fetchFromGitHub {
+    owner = "BubblesInTheTub";
+    repo = pname;
+    rev = "c982324b0af6094e7e28b106f50cb8bbec70d227";
+    sha256 = "+fSkB+u/CZ6VT+q8E0Ulvcr+BFVcxN4rCF4F0IyBFow=";
+  };
+
+  buildInputs = [ ncurses.dev ];
```

This is implied by buildInputs
```suggestion
  buildInputs = [ ncurses ];
```